src/main/java/ru/ayaz/User.java

```diff
@@ -0,0 +1,23 @@
+package ru.ayaz;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class User {
+
+    private String nickname;
+    private Map<String, Video> userVideos;
```

Comment:
&#x1F34E; **Issue**
I don't see the need for using map for videos. For now you can do just fine with List or Set. No need to complicate things ahead of time.

I would also get rid of "user" prefix here as well.